skimage/registration/_phase_cross_correlation.py

```diff
@@ -4,7 +4,8 @@
 """
 
 import numpy as np
-from .._shared.fft import fftmodule as fft
+# use scipy FFTs to preserve dtype (numpy enforces float64)
+from scipy.fftpack import fftn, ifftn, fftfreq
```

Comment:
Shouldn't we use the same `try .. except ...` block as in `_masked_phase_cross_correlation`?

Reply:
Yes, I was testing to make sure the tests passed with fftpack and forgot go back to the try/except form afterward